r/work.R

```diff
@@ -41,7 +41,33 @@ run_post <- function(req, post_fun, data) {
     debug_step <- Sys.getenv("DEBUG_STEP", unset = "")
 
     handle_debug(req, debug_step)
-    post_fun(req, data)
+
+    message(rep("✧",100))
+    message("➥ Starting ",sub("run","",basename(req$path)))
+    message("Input:")
+    message(str(req$body))
+
+    tryCatch({
+            message("\nSeurat logs:")
+            message("➡️ \n")
+            tstart <- Sys.time()
+            res <- post_fun(req, data)
+            message("\n⬅️")
+
+            message("\nResult head: ")
```

Comment:
apart from the result head, do you think perhaps knowing the size of the result might help too?

Reply:
The problem with that one is that it might not work for all worker functions, but it's a good one

Reply:
Added